examples/with-swc/src/preconditions/OwnerOnly.ts

```diff
@@ -1,6 +1,6 @@
 import { Precondition } from '@sapphire/framework';
 import type { Message } from 'discord.js';
-import { envParseArray } from '../lib/env-parser';
+import { envParseArray } from '#lib/env-parser';
```

Comment:
```suggestion
import { envParseArray } from '../lib/env-parser';
```

This project hasn't been set up to have subpath imports